evopy/individual.py

```diff
@@ -38,3 +45,47 @@ def evaluate(self, fitness_function):
         self.fitness = fitness_function(self.genotype)
 
         return self.fitness
+
+    def _reproduce_single_variance(self):
+        """Create a single offspring individual from the set genotype and strategy parameters.
+
+        This function uses the single variance strategy.
+
+        :return: an individual which is the offspring of the current instance
+        """
+        new_genotype = self.genotype + \
+                       self.strategy_parameters[0] * np.random.randn(self.length)
+        scale_factor = np.random.randn() / (2 * self.length)
```

Comment:
I think we are sampling from the normal distribution the wrong way. Numpy's documentation for randn suggests the following:
> For random samples from N(mu, sigma^2), use:
> `sigma * np.random.randn(...) + mu`

We are multiplying with the variance and not the standard deviation. To get the correct result we should probably divide by the square root of `2 * self.length`. We do this multiple times in this class and perhaps even in other classes, so we should double check all places where we use randn...

Reply:
I think you're right, I've fixed this